### PR TITLE
Improve vim plugin mentions in readme

### PR DESCRIPTION
**Pull request by subnut**
New neovim plugin incorporated. 
Difference between different Vim plugins highlighted in **bold**

Extra whitespaces removed

### CODE REVIEW

**Comment by subnut** [comment]
Ping @fregante

**Comment by fregante** [comment]
Thanks for this! However it gives too much attention to vim stuff. We to either revert the style and just keep the best 2 solutions or wrap everything in a `details` tag so that when it’s closed it only takes 1 line.

**Comment by subnut** [comment]
I wrapped up all vim-related plugins inside a `<details>` tag.
Now is it OK?

File: README.md [hunk 1]
```diff
@@ -12,17 +12,23 @@ Use your text editor to write in your browser. Everything you type in the editor
...
-    + [**Neovim** plugin](https://github.com/raghur/vim-ghost) (Third party)
     + [**Emacs** package](https://melpa.org/#/atomic-chrome) - [Repo](https://github.com/alpha22jp/atomic-chrome) (Third party)
     + [**Acme** client](https://github.com/fhs/Ghost) (Third party)
+    + <details>
+        <summary><b>Vim</b>/<b>Neovim</b> (Third party)</summary>
+        <ul>
+          <li><a href="https://github.com/falstro/ghost-text-vim">Tcl script for Vim</a>
+          <li><a href="https://github.com/pandysong/ghost-text.vim">Python plugin for Vim</a>
+          <li><a href="https://github.com/raghur/vim-ghost">Python plugin for Vim/Neovim</a>
+          <li><a href="https://github.com/subnut/nvim-ghost.nvim">Native Neovim plugin</a>
```
Comment (fregante) [inline README.md:23]: Are these descriptions correct? Does "native" make sense for a neovim plugin written in vim?

Comment (fregante) [reply README.md:23]: This is even clearer and follows the others (they all start with the editor’s name)

```suggestion
          <li><a href="https://github.com/falstro/ghost-text-vim">Vim (written in Tcl)</a>
          <li><a href="https://github.com/pandysong/ghost-text.vim">Vim (written in Python)</a>
          <li><a href="https://github.com/raghur/vim-ghost">Vim/Neovim (written in Python) </a>
          <li><a href="https://github.com/subnut/nvim-ghost.nvim">Neovim (written in Vim)</a>
```

**Comment by subnut** [comment]
No no, it is of no use for the end-user to know which language a plugin is written in. The end user only needs to know about what it's dependencies are. Why? Because satisfying dependencies can be a pain in the a**.

**Comment by fregante** [comment]
Isn’t the fact that it’s written in Python or TLC imply that they require python or tlc installed?

**Comment by fregante** [comment]
I understand that you wrote a better dependency-free plugin and I appreciate your enthusiasm, but here I just need to list a few options and move on, I don’t need detailed explanations of each. 

What if I use this format:

- Vim plugin (requires Python)
- Neovim plugin (no dependencies)

**Comment by subnut** [comment]
> Isn’t the fact that it’s written in Python or TLC imply that they require python or tlc installed?

Nope! In fact, mine is mainly written in python

**Comment by subnut** [comment]
> I understand that you wrote a better dependency-free plugin and I appreciate your enthusiasm, but here I just need to list a few options and move on, I don’t need detailed explanations of each because those details can change.

Oh, right. So it should be compact, right?
Hmm... I have given a compact layout in the latest commit. See if that is OK

**Comment by fregante** [comment]
> Nope! In fact, mine is mainly written in python

So how can it run without python?

**Comment by subnut** [comment]
> So how can it run without python?

I pre-compile it using pyinstaller. The user downloads the binary pre-compiled, and the binary itself does all the hard work.

File: README.md [hunk 2]
```diff
@@ -12,17 +12,23 @@ Use your text editor to write in your browser. Everything you type in the editor
...
-    + [**Vim** script](https://github.com/falstro/ghost-text-vim) (Third party)
-    + [**Vim** plugin](https://github.com/pandysong/ghost-text.vim) (Third party)
-    + [**Neovim** plugin](https://github.com/raghur/vim-ghost) (Third party)
     + [**Emacs** package](https://melpa.org/#/atomic-chrome) - [Repo](https://github.com/alpha22jp/atomic-chrome) (Third party)
     + [**Acme** client](https://github.com/fhs/Ghost) (Third party)
+    + <details>
+        <summary><b>Vim</b>/<b>Neovim</b> (Third party)</summary>
+        <ul>
+          <li><a href="https://github.com/subnut/nvim-ghost.nvim"><b>Neovim</b></a>
+          <li><a href="https://github.com/raghur/vim-ghost"><b>Vim</b> (<tt>+python3</tt>) & <b>Neovim</b> (<tt>pynvim</tt>)</a>
```
Comment (fregante) [inline README.md:21]: Is a `+` missing here?
```suggestion
          <li><a href="https://github.com/raghur/vim-ghost"><b>Vim</b> (<tt>+python3</tt>) & <b>Neovim</b> (<tt>+pynvim</tt>)</a>
```

Comment (subnut) [reply README.md:21]: No. The things beginning with `+` are compilation flags.
`pynvim` is a python package, installed using `pip` (pip is to python what npm is to node)

**Comment by subnut** [comment]
@fregante Is the current description concise enough?

**Review by fregante — APPROVED** [review]

**Comment by subnut** [comment]
:+1: 

Let's merge it?

**Comment by subnut** [comment]
@fregante Is it OK? or are changes required?

**Pull request merged**